Fix version number, cuz it's confusing, when help menu shows Firefox's version, while package has Waterfox's version...

diff --git a/browser/config/version_display.txt b/browser/config/version_display.txt
index f31fb0732512..f3b54ae0a9be 100644
--- a/browser/config/version_display.txt
+++ b/browser/config/version_display.txt
@@ -1 +1 @@
-102.4.0esr
+G5.0.2
-- 
2.38.0

